.github/CONTRIBUTING.md

```diff
@@ -1,65 +1,55 @@
 NewPipe contribution guidelines
 ===============================
 
-PLEASE READ THESE GUIDELINES CAREFULLY BEFORE ANY CONTRIBUTION!
-
 ## Crash reporting
 
-Do not report crashes in the GitHub issue tracker. NewPipe has an automated crash report system that will ask you to
-send a report via e-mail when a crash occurs. This contains all the data we need for debugging, and allows you to even
-add a comment to it. You'll see exactly what is sent, the system is 100% transparent.
+Report crashes in the automated crash report system of NewPipe instead of the GitHub issue tracker.
+Reports are sent via e-mail, and contain all data needed for debugging, including your comments.
+You'll see exactly what is sent, and then send it.
```

Comment:
@Stypox @wb9688 @B0pol that part is somehow outdated as we already have an in-app button for reporting on GutHub.